ros_ws/src/control/include/control/remote_control.h

```diff
@@ -0,0 +1,99 @@
+/**
+ * @file RemoteControl.h
+ * @class RemoteControl
+ * @brief The class which converts joy messages to twist
+ *
+ * This class will act as a way to take controller inputs and output them to a format which is usable by the robot.  It
+ * is currently setup to accept joy messages and output them as a twist message which only sets the linear.x and
+ * angular.z velocities.
+ */
+
+#ifndef CONTROL_INCLUDE_CONTROL_REMOTE_CONTROL_H_
+#define CONTROL_INCLUDE_CONTROL_REMOTE_CONTROL_H_
+
+#include <ros/ros.h>
+#include <sensor_msgs/Joy.h>
+#include <geometry_msgs/Twist.h>
+
+#define Y_AXIS_L_STICK 1
+#define X_AXIS_R_STICK 2
+
+/**
+ * @brief The topic that joy messages will be received from
+ */
+extern const std::string JOY_TOPIC;
+/**
+ * @brief The topic that twist messages will be published on
+ */
+extern const std::string TELEOP_TOPIC;
```

Comment:
I am generally apprehensive of declaring constants in the global namespace. This could potentially cause name collisions in the future. How I would rather handle this is by either scoping these constants into a "constants" namespace (or similar), or create a class for constants and collecting these in there. I'm leaning more towards the second option, but first I'm curious to know what you think.